main.tf

```diff
@@ -367,6 +367,23 @@ resource "aws_codebuild_project" "default" {
     }
   }
 
+  dynamic "secondary_sources" {
+    iterator = secondary_source
+    for_each = length(var.secondary_sources) > 0 ? var.secondary_sources : []
```

Comment:
you can delete the condition and it will null if secondary sources is empty

Reply:
Changed pushed, is this what you mean ?

Reply:
Done